gix-filter/tests/fixtures/baseline.sh

```diff
@@ -17,7 +17,7 @@ function repo_assertions() {
 
   git config filter.arrow.clean "$driver clean %f"
   git config filter.arrow.smudge "$driver smudge %f"
-  git config filter.arrow.requred true
+  git config filter.arrow.required true
```

Comment:
That's a good one! It's also interesting that the test doesn't seem to care about it either way.